### PR TITLE
Rasterize url bug

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-44315

## Description
Fixed an issue where the ***rasterize*** command failed to parse URLs.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by adi88d — APPROVED** [review]
nice!

**Review by tkatzir — APPROVED** [review]

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 1]
```diff
@@ -1127,6 +1127,8 @@ def add_filename_suffix(file_names: list, file_extension: str):
 
 def rasterize_command():  # pragma: no cover
     urls = demisto.getArg('url')
+    # Do not remove this line
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:1130]:
```suggestion
    # Do not remove this line, as rasterize does not support array in `url`.
```

**Review by tkatzir — APPROVED** [review]
Approved, with a comment

File: Packs/rasterize/ReleaseNotes/2_0_30.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Fixed an issue where the ***rasterize*** command failed to parse URLs.
```
Comment (tkatzir) [inline Packs/rasterize/ReleaseNotes/2_0_30.md:6]:
```suggestion
- Fixed an issue where the ***rasterize*** command failed to parse URLs that contained a comma.
```

**Review by tkatzir — CHANGES_REQUESTED** [review]
Need to remove `isArray: true` in the yml file.

**Review by tkatzir — APPROVED** [review]
Great! Approved.

**Review by ilaredo — APPROVED** [review]
line 877:   paths = argToList(path)
become redundant

**Comment by merit-maita** [comment]
the rasterize_test TPB  failing in saas is a known issue, so it's going to be force merged.

**Pull request merged**